pkgs/development/tools/gertty/default.nix

```diff
@@ -0,0 +1,48 @@
+{ python3, python3Packages, stdenv }:
```

Comment:
- You are using both `python3Packages` and `python3.pkgs`, which are equivalent, you can get rid of one of them.
- You can directly make use of `lib` if you only need that one instead of `stdenv.lib`

Reply:
indeed, thanks